vowpalwabbit/core/src/reductions/eigen_memory_tree.cc

```diff
@@ -721,6 +880,10 @@ size_t read_model_field(io_buf& io, reductions::eigen_memory_tree::emt_tree& tre
   bytes += read_model_field(io, router_type);
   tree.router_type = static_cast<reductions::eigen_memory_tree::emt_router_type>(router_type);
 
+  uint32_t initial_type{};
+  bytes += read_model_field(io, initial_type);
+  tree.initial_type = static_cast<reductions::eigen_memory_tree::emt_initial_type>(initial_type);
+
```

Comment:
You don't need to save it like this since its a command line option, you can just use .keep() on the option

Reply:
Oh that's great to know. I've actually always wondered what `.keep()` did. I'll update the code.

Reply:
It has been removed (along with several other EMT parameters that could be removed from save/load due to .keep). 

I also added checks to the save load unittest to for all parameters I removed from explicit saving and loading.